docs/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -euf -o pipefail
+gitbook build
+rm -rf ../docs
```

Comment:
is it safe to always remove the directory?

Reply:
it is... because `/docs` is auto-generated and there's git to track history.

Reply:
yeah, it's auto-generated here, but it is quite a established rules to put docs under `docs` directory.